CPP/SortingAlgrothims/Sort.cpp

```diff
@@ -72,6 +72,41 @@ void SortingAlgrothim<T>::InsertionSort(T arr[], uint32_t Size, SortingType_t So
         arr[j+1] = key;
     }
 }
+template<typename T>
+void SortingAlgrothim<T>::CountingSort(T arr[], uint32_t size,uint32_t Range, SortingType_t SortingType )
+{   Range++;
+    T Count_Arr[Range]={0};
```

Comment:
This is invalid even if it work. If you don't know the array size in compilation time (i.e. known only in run-time) then you have to use `vector` which is dynamic allocated array